guide/.vuepress/sidebar.ts

```diff
@@ -53,6 +53,14 @@ export default {
 				'/creating-your-bot/event-handling.md',
 			],
 		},
+		{
+			text: 'Additional Features',
+			children: [
+				'/additional-features/',
```

Comment:
```suggestion
```
u dont have index file here

Reply:
there's no index file in the `creating-your-bot` folder too

Reply:
It does, vuepress treats README as index files